docs/commitstyle.md

```diff
@@ -0,0 +1,47 @@
+# Commit Style Guidelines for Kiwix-Android
+
+### Message Structure
+Commit messages consist of three distinct parts, separated by a blank line: the title, an optional body/content, and an optional footer/metadata. The layout looks like this:
+
+type: subject
+
+body
+
+***
+
+### Title
+The title consists of the subject and type of the commit message.
+
+### Type
+The type is contained within the title and can be one of the following types:
```

Comment:
What are the benefits of these that aren't included in the issue labels?

Reply:
The issue labels are abstract. These one's explicitly show, what has been done in the commit. If you want i can also add these to the issue labels.

Reply:
In the past I have normally just used an imperative verb such as Fix, Improve, Implement, Remove. I am not sure what limiting our variety of these verbs adds.

Reply:
Okay. I think any thing that makes the intention of the commit clear works. What do you want me to add in here ?

Reply:
I would maybe put something like.

"Imperative verb (not past tense) such as Fix, Implement, Remove, Duplicate... describing the main actions performed by this commit"